src/pages/GetStarted/CustomWorkspaceTab/DevfileSelector.module.css

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Please create directory `./DevfileSelector` and move there `DevfileSelector.tsx` and `DefvileSelector.module.css`, rename those files to be `index.tsx` and `index.module.css` respectively.

And repeat steps above for `InfrastructureNamespace`, `StorageType` and `WorkspaceName`.

Reply:
done

Reply:
You've only done the fist step. 
Now please rename `DevfileSelector.tsx` to `index.tsx` and `DevfileSelector.module.css` to `index.module.css`.
Then update parent component import to point to the `DevfileSelector` directory.

And do the same for the rest of components I mentioned.

Reply:
done